docs/external_rules.rst

```diff
@@ -66,3 +66,47 @@ Rules can have configurable values. You need to specify them in rule body after
 Configurable parameter can be referred by its :code:`public_name` in command line options::
 
     robocop --ext-rules my/own/rule.py --configure dummy-in-name:public_name:AnotherDummy
+
+Import from external module
+----------------------------
+Robocop rules can be written in separate, distributed module. For example using ``RobocopRules`` module name and following
+directory structure::
+
+    RobocopRules/
+    RobocopRules/__init__.py
+    RobocopRules/some_rules.py
+    setup.py
+
+inside ``__init__.py``::
+
+    from .some_rules import CustomRule
+
+    all = ['CustomRule']
```

Comment:
To comply with the class defined in `RobocopRules/some_rules.py`:

```suggestion
    from .some_rules import MissingKeywordName

    all = ['MissingKeywordName']
```

Reply:
Good eye! But it should be other way around - since MissingKeywordName checker already exist in default ones, I should rename class to CustomRule.

Reply:
Good point!